src/main/java/fi/aalto/cs/apluscourses/intellij/actions/SubmitExerciseAction.java

```diff
@@ -155,6 +155,10 @@ public void update(@NotNull AnActionEvent e) {
       e.getPresentation().setEnabled(project != null
               && authentication != null && courseViewModel != null
               && (isSubmittableExerciseSelected || isSubmittableSubmissionSelected));
+      var selectedEx = exercisesViewModel.findSelected().getLevel(2);
+      var isTutorial = selectedEx instanceof ExerciseViewModel
+          && ExerciseViewModel.Status.TUTORIAL.equals(((ExerciseViewModel) selectedEx).getStatus());
```

Comment:
Nit: There could be something like `isSubmittable` which would then be overridden by tutorial exercises. But in the future, the tutorials too should be submittable so I guess this is just a temporary issue.